Gemfile.lock

```diff
@@ -391,8 +410,5 @@ DEPENDENCIES
   web-console (>= 3.3.0)
   webmock
 
-RUBY VERSION
```

Comment:
This removal is from the bind mount ... I'm assuming since we are not using `rbenv` on the container.